SD-K9/e2e/src/app.e2e-spec.ts

```diff
@@ -1,14 +1,34 @@
+import { browser, element, by } from "protractor"; 
 import { AppPage } from './app.po';
+import { HomePage } from './home.po';
 
-describe('new App', () => {
-  let page: AppPage;
+describe("Directions", () => {
+  const home = new HomePage();
 
   beforeEach(() => {
-    page = new AppPage();
+    home.load();
   });
 
-  it('should be blank', () => {
-    page.navigateTo();
-    expect(page.getParagraphText()).toContain('The world is your oyster.');
+  describe("after launching the application", () => {
+    it("displays the home page", () => {
+      expect(home.rootElement().isPresent()).toEqual(true);
+    });
+
+    it("the page contains an area prompting user to enter starting point", () => {
```

Comment:
Does this have a naming convention. For instance, in unit test, you specify the function name by 
`#FuntionName should...`